spec/controllers/resources_controller/nested_controller/double_nested_controller.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
```

Comment:
Is it possible that you've missed to add the `_spec.js` at the name of this file?
`double_nested_controller_spec.js` instead of `double_nested_controller.js`

Reply:
nop, this is not a test, it's just a controller